src/components/HomePageLogo.tsx

```diff
@@ -1,12 +1,28 @@
-import React from 'react';
+import React, { JSX } from 'react';
+import { ThemeDataType, ThemeType } from 'src/themes';
+import { useTheme } from 'styled-components';
 import { LogoContainer } from './HomePageLogo.style';
 import { ArDriveLogoLight } from './icons/ArDriveLogoLight';
+import { ArDriveLogoDark } from './icons/ArDriveLogoDark';
 
 export function HomePageLogo(): JSX.Element {
+	const themeData = useTheme() as unknown as ThemeDataType;
```

Comment:
I think you can remove 'as unknown' and just have 'as ThemeDataType'. (Worked here when I tried it.)

Reply:
Ahh yes! I removed the casting completely because earlier I made the DefaultTheme to be ThemeDataType.
Applied - 2a39310a2a5b6fda2f627c2ed38afcca0d36cf81